StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/ClickableLabelNew.swift

```diff
@@ -0,0 +1,174 @@
+        linkColor: UIColor = .textBrand,
+        alignCenter: Bool = false
+    ) {
+        self.font = font
+        self.boldFont = boldFont
+        self.linkFont = linkFont
+        self.textColor = textColor
+        self.alignCenter = alignCenter
+        super.init(frame: .zero)
+        textView.isScrollEnabled = false
+        textView.delaysContentTouches = false
+        textView.isEditable = false
+        textView.isSelectable = true
+        textView.backgroundColor = UIColor.clear
+        // Get rid of the extra padding added by default to UITextViews
+        textView.textContainerInset = .zero
+        textView.textContainer.lineFragmentPadding = 0.0
+        textView.linkTextAttributes = [
+            .foregroundColor: linkColor
+        ]
+        textView.delegate = self
+        // remove clipping so when user selects an attributed
+        // link, the selection area does not get clipped
+        textView.clipsToBounds = false
+        addAndPinSubview(textView)
+
+        // enable faster tap recognizing
+        if let gestureRecognizers = textView.gestureRecognizers {
+            for gestureRecognizer in gestureRecognizers {
+                if let tapGestureRecognizer = gestureRecognizer as? UITapGestureRecognizer,
+                    tapGestureRecognizer.numberOfTapsRequired == 2
+                {
+                    // double-tap gesture recognizer causes a delay
+                    // to single-tap gesture recognizer so we
+                    // disable it
+                    tapGestureRecognizer.isEnabled = false
+                }
+            }
+        }
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    /// Helper that automatically handles extracting links and, optionally, opening it via `SFSafariViewController`
+    @available(iOSApplicationExtension, unavailable)
+    func setText(
+        _ text: String,
+        action: @escaping ((URL) -> Void) = { url in
+            SFSafariViewController.present(url: url)
+        }
+    ) {
+        let textLinks = text.extractLinks()
+        setText(
+            textLinks.linklessString,
+            links: textLinks.links.map {
+                ClickableLabelNew.LinkDescriptor(
+                    range: $0.range,
+                    urlString: $0.urlString,
+                    action: action
+                )
+            }
+        )
+    }
+
+    private func setText(
+        _ text: String,
+        links: [LinkDescriptor]
+    ) {
+        let paragraphStyle = NSMutableParagraphStyle()
+        if alignCenter {
+            paragraphStyle.alignment = .center
+        }
+        paragraphStyle.minimumLineHeight = font.lineHeight
+        paragraphStyle.maximumLineHeight = font.lineHeight
+        let string = NSMutableAttributedString(
+            string: text,
+            attributes: [
+                .paragraphStyle: paragraphStyle,
+                .font: font.uiFont,
```

Comment:
Key danger to look out in this code that we are unwrapping `uiFont`. 

Compiler will happily (but wrongly) accept `FinancialConnectionsFont` instead of what it should be (`UIFont`)